datapreview.tex

```diff
@@ -1,19 +1,22 @@
 \section{Data Previews and Data Release 1} 
 \label{sec:datapreview}
 
-A series of three Data Previews (DP) are planned  based on commissioning data to support the community as they develop their LSST analysis software and worfklows, and to enable high-impact science as soon as possible.
+A series of three Data Previews (DP) are planned  based on commissioning data to support the community as they develop their LSST analysis software and workflows, and to enable high-impact science as soon as possible.
 \begin{itemize}
 \item Data Preview 0 (DP0): Based on simulated LSST-like data.
-\item Data Preview 1 (DP1): Based on a subset of early science-grade commissioning data taken with LSSTCam.
+\item Data Preview 1 (DP1): Based on a subset of early science-grade commissioning data taken with either ComCam or LSSTCam.
 \item Data Preview 2 (DP2): Based on a full reprocessing of all science-grade LSSTCam data taken during commissioning.
 \end{itemize}
 
-Due to the relatively short time periods available for commissioning observations (\S~\ref{ssec:scenarios}), these Data Previews will necessarily be limited in their area and temporal coverage relative to full a Data Release, however all Data Preview data products will be in the same science data model format as for future Data Releases.
+Due to the relatively short time periods available for commissioning observations (\S~\ref{ssec:scenarios}), these Data Previews will necessarily be limited in their area and temporal coverage relative to full a Data Release, however all Data Preview data products will be in the same science data model format as for future LSST Data Releases.
 
 The data products that comprise a Data Preview are produced by the LSST Science Pipelines \citep{2019ASPC..523..521B,2018PASJ...70S...5B}.
 For an introduction to the LSST data products, see \citet{RubinDataProductsAbridged} and for a detailed description, see the LSST \dpdd{},  \citedsp{LSE-163}.
 Each pre-operations Data Preview and survey Data Release will be accompanied by its own release-specific DPDD\footnote{For an example data release DPDD, see the online DP0.2 documentation {\url{https://dp0-2.lsst.io/data-products-dp0-2/}}.}, giving e.g. the  database schema for the catalogs included in that dataset.
```

Comment:
I would remove "pre-operations", as it could be read to exclude DP2, which is surely not what was intended.